.ci_support/linux_64_.yaml

```
@@ -15,7 +19,7 @@ cxx_compiler_version:
 docker_image:
 - quay.io/condaforge/linux-anvil-cos7-x86_64
 libopenvino_dev:
-- 2023.3.0
+- 2024.0.0
```

```suggestion
- 2024.1.0
```
and in other CI support files as well